src/stdlib_linalg.fypp

```diff
@@ -80,8 +88,93 @@ module stdlib_linalg
     #:endfor
   end interface outer_product
 
+
+  ! Check for squareness
+  interface is_square
+    !! version: experimental
+    !!
+    !! Checks if a matrix (rank-2 array) is square
+    !! ([Specification](../page/specs/stdlib_linalg.html#description_4))
+    #:for k1, t1 in RCI_KINDS_TYPES
+      module procedure is_square_${t1[0]}$${k1}$
+    #:endfor
+  end interface is_square
+
+
+  ! Check for diagonality
+  interface is_diagonal
+    !! version: experimental
+    !!
+    !! Checks if a matrix (rank-2 array) is diagonal
+    !! ([Specification](../page/specs/stdlib_linalg.html#description_5))
+    #:for k1, t1 in RCI_KINDS_TYPES
+      module procedure is_diagonal_${t1[0]}$${k1}$
+    #:endfor
+  end interface is_diagonal
+
+
+  ! Check for symmetry
+  interface is_symmetric
+    !! version: experimental
+    !!
+    !! Checks if a matrix (rank-2 array) is symmetric
+    !! ([Specification](../page/specs/stdlib_linalg.html#description_6))
+    #:for k1, t1 in RCI_KINDS_TYPES
+      module procedure is_symmetric_${t1[0]}$${k1}$
+    #:endfor
+  end interface is_symmetric
+
+
+  ! Check for skew-symmetry
+  interface is_skew_symmetric
+    !! version: experimental
+    !!
+    !! Checks if a matrix (rank-2 array) is skew-symmetric
+    !! ([Specification](../page/specs/stdlib_linalg.html#description_7))
+    #:for k1, t1 in RCI_KINDS_TYPES
+      module procedure is_skew_symmetric_${t1[0]}$${k1}$
+    #:endfor
+  end interface is_skew_symmetric
+
+
+  ! Check for Hermiticity
```

Comment:
```suggestion
  ! Check for Hermicity
```

Reply:
Interesting, the only version I had heard was "Hermiticity", which looks common enough in [quantum mechanics](https://aip.scitation.org/doi/10.1063/1.1418246), but is very near to "hermeticity" (being hermetically sealed).
But "Hermicity" appears to be the only one with a [wiktionary page](https://en.wiktionary.org/wiki/hermicity) and seems more common in [mathematics](https://www.researchgate.net/publication/334325996_Cramer%27s_rules_for_the_system_of_quaternion_matrix_equations_with_e_-Hermicity).
I'm a bit divided, so I decided to check the [worldwide popularity](https://trends.google.com/trends/explore?date=all&q=hermicity,hermiticity) which shows that "Hermiticity" is slightly more common.
I could still be convinced either way though.

Reply:
Let's keep it with "hermiticity" in that case. Just for the record, that wiktionary also has a [hermiticity](https://en.wiktionary.org/wiki/hermiticity#English) page.